### PR TITLE
LPS-148182 [CP 2.0] Analytics Cloud Setup Style

### PR DESCRIPTION
**Pull request by claricebrasil**
### How it was developed?
It was created a form style to update Analytics Cloud status using the Formik library to use for email, data center location, workspace URL, and timezone validations. Also, Formik facilitate with helpers text of each field.

### What has been affected by the development?
We had to create a new route to have access to Analytics Cloud page both in the side menu and to have an friendly URL to easily share this page.

### CODE REVIEW

**Review by Lucas2691 — APPROVED** [review]
Nice job.

**Review by igordmf — APPROVED** [review]
Good job!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+const AdminInputs = ({admin, id}) => {
+	const debouncedEmail = useDebounce(admin?.email, 500);
+	const [bannedDomain, setBannedDomain] = useState(debouncedEmail);
+
+	const [fetchBannedDomain, {data}] = useLazyQuery(getBannedEmailDomains);
+	const bannedDomainsItems = data?.c?.bannedEmailDomains?.items;
+
+	useEffect(() => {
+		const emailDomain = debouncedEmail.split('@')[1];
```
Comment (iagochaves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js:28]:
```suggestion
		const [, emailDomain] = debouncedEmail.split('@');
```

**Review by iagochaves — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js (hunk 1 above)
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js:28]: Done, thank you!!

**Review by iagochaves — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+import {useLazyQuery} from '@apollo/client';
+import ClayForm, {ClayInput} from '@clayui/form';
+import {useEffect, useState} from 'react';
+import {Input} from '../../../../components';
+import useDebounce from '../../../../hooks/useDebounce';
+import {getBannedEmailDomains} from '../../../../services/liferay/graphql/queries';
+import {isValidEmail} from '../../../../utils/validations.form';
+
+const AdminInputs = ({admin, id}) => {
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js:20]: should be admin the right name for this component?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+			</ClayInput.Group>
+
+			<Input
+				groupStyle="mb-0"
+				label="System Admin’s Github Username"
+				name={`dxp.admins[${id}].github`}
+				required
+				type="text"
+			/>
+		</ClayForm.Group>
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js:85]: There are somes inputs about DXP Cloud form here , but the folder is SetupAnalyticsCloudForm

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js (hunk 2 above)
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js:20]: Yeah, this file doesn't exist anymore. I'm going to push again. Thank you!!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js (hunk 3 above)
Comment (claricebrasil) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupAnalyticsCloudForm/AdminInputs/index.js:85]: Same here, this file with this elements doesn't exists anymore.

**Comment by claricebrasil** [comment]
@jairmedeiros I believe everything is ok now.

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by jairmedeiros** [comment]
ci:forward

**Comment by iagochaves** [comment]
ci:forward

**Pull request closed without merging**